### PR TITLE
Email the admin about any changes in the organization's data use permissions

### PR DESCRIPTION
**Pull request by dimasciput**
For https://github.com/kartoza/sawps/issues/1894

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 9 min 40 sec

File: django_project/frontend/views/organisations.py [hunk 1]
```diff
@@ -52,6 +90,31 @@ def save_permissions(request, organisation_id):
...
+                        'organisation': organisation.name,
+                        'support_email': SUPPORT_EMAIL,
+                        'changes_list': changes_list_html
+                    }
+                )
+                send_mail(
+                    subject,
+                    None,
+                    settings.SERVER_EMAIL,
+                    list(User.objects.filter(
```
Comment (danangmassandy) [inline django_project/frontend/views/organisations.py:112]: i think it's better to add is_active=True to the filter of the user queryset

**Review by danangmassandy — COMMENTED** [review]
do you think that we should also send the email to other manager in this organisation? I'll confirm with @amyburness

**Comment by danangmassandy** [comment]
confirmed with Amy, it's only superadmin that should get notified.

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 9 min 48 sec

**Review by danangmassandy — APPROVED** [review]

**Comment by danangmassandy** [comment]
@dimasciput I've updated the filter.

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 3 sec

**Pull request merged**